driverless/driverless/src/main/java/com/upgrad/hirewheels/entities/Users.java

```diff
@@ -23,7 +23,7 @@
     @Column( nullable = false, unique = true)
     String email;
```

Comment:
You can change the name of the class to User.java. This won't have any implication because of Oracle because you are explicitly setting the table name using @Table annotation in this class.